libs/designsystem/src/lib/components/grid/grid.component.ts

```diff
@@ -5,6 +5,9 @@ import { ScssHelper } from '../../scss/scss-helper';
 import { BreakpointHelperService } from './breakpoint-helper.service';
 import { GridCardConfiguration } from './grid-card-configuration';
 
+const GRID_DEPRICATION_WARNING =
+  'Deprecation warning: The "kirby-grid" component will be removed in a future release of Kirby designsystem. Consider using kirby-grid, kirby-grid-item css-classes as an alternative solution.';
```

Comment:
Let us link to grid docs here too (https://cookbook.kirby.design/#/home/showcase/grid)